iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/FileUploadTests.java

```diff
@@ -284,22 +284,6 @@ private void verifyNotification(FileUploadNotification fileUploadNotification, F
         fileUploadState.fileUploadNotificationReceived = SUCCESS;
     }
 
-    @Test (timeout = MAX_MILLISECS_TIMEOUT_KILL_TEST)
-    @ContinuousIntegrationTest
-    public void uploadToBlobAsyncSingleFileZeroLength() throws URISyntaxException, IOException, InterruptedException, IotHubException, GeneralSecurityException
```

Comment:
the uploadToBlobAsync API was removed from the device client

Reply:
We should update the msdn docs for file upload to point to new code instead of deprecated code. Probably need to do this where ever applicable. This is not related to this PR but would be good to create a task for that.